third_party/pyth/multisig-wh-message-builder/src/index.ts

```diff
@@ -0,0 +1,284 @@
+
+  console.log("Creating wormhole instructions...");
+  const wormholeIxs = await getWormholeMessageIx(
+    cluster,
+    emitter,
+    emitter,
+    message.publicKey,
+    squads.connection,
+    payload
+  );
+  console.log("Wormhole instructions created.");
+
+  console.log("Creating transaction...");
+  // transfer sol to the message account
+  await squads.addInstruction(newTx.publicKey, wormholeIxs[0]);
+  // wormhole post message ix
+  await squads.addInstruction(newTx.publicKey, wormholeIxs[1]);
+
+  await squads.activateTransaction(newTx.publicKey);
+  console.log("Transaction created.");
+}
+
+async function executeMultisigTx(
+  cluster: string,
+  emitter: PublicKey,
+  walletPath: string,
+  messagePath: string,
+  txPDA: PublicKey,
+  rpcUrl: string
+) {
+  const wallet = new NodeWallet(
+    Keypair.fromSecretKey(
+      Uint8Array.from(JSON.parse(fs.readFileSync(walletPath, "ascii")))
+    )
+  );
+  console.log(`Loaded wallet with address: ${wallet.publicKey.toBase58()}`);
+
+  const message = Keypair.fromSecretKey(
+    Uint8Array.from(JSON.parse(fs.readFileSync(messagePath, "ascii")))
+  );
+  console.log(
+    `Loaded message account with address: ${message.publicKey.toBase58()}`
+  );
+
+  const squads =
+    cluster === "devnet" ? Squads.devnet(wallet) : Squads.mainnet(wallet);
+
+  const executeIx = await squads.buildExecuteTransaction(
+    txPDA,
+    wallet.publicKey
+  );
+  executeIx.keys.forEach((key) => {
+    if (key.pubkey.equals(message.publicKey)) {
+      key.isSigner = true;
+    }
+  });
+
+  // airdrop 0.1 SOL to emitter if on devnet
+  if (cluster === "devnet") {
+    const airdropSignature = await squads.connection.requestAirdrop(
+      emitter,
+      0.1 * LAMPORTS_PER_SOL
+    );
+    const { blockhash, lastValidBlockHeight } =
+      await squads.connection.getLatestBlockhash();
+    await squads.connection.confirmTransaction({
+      blockhash,
+      lastValidBlockHeight,
+      signature: airdropSignature,
+    });
+    console.log("Airdropped 0.1 SOL to emitter");
+  }
+
+  const { blockhash, lastValidBlockHeight } =
+    await squads.connection.getLatestBlockhash();
+  const executeTx = new anchor.web3.Transaction({
+    blockhash,
+    lastValidBlockHeight,
+    feePayer: wallet.payer.publicKey,
+  });
+  const provider = new anchor.AnchorProvider(squads.connection, wallet, {
+    ...anchor.AnchorProvider.defaultOptions(),
+    commitment: "confirmed",
+    preflightCommitment: "confirmed",
+  });
+  executeTx.add(executeIx);
+
+  const signature = await provider.sendAndConfirm(executeTx, [
+    wallet.payer,
+    message,
+  ]);
+  console.log(
+    `Executed tx: https://explorer.solana.com/tx/${signature}${
+      cluster === "devnet" ? "?cluster=devnet" : ""
+    }`
+  );
+
+  const txDetails = await squads.connection.getParsedTransaction(
+    "43edaJsdeYu7ZceTCWC9S7GzVwp67yVfRXgYb9TVWuHEoiWFJo95QawA8W9iWr64XspbaP7SmVjNL3mqG16KD2B4"
```

Comment:
Magic pubkey?

Reply:
forgot to change back while troubleshooting 🤦🏻‍♂️ thanks for the catch